### PR TITLE
Add 2020 Linux foundation

### PR DESCRIPTION
**Pull request by pranshukharkwal**
@vsvipul

### CODE REVIEW

File: gsoc/index.html [hunk 1]
```diff
@@ -518,7 +522,8 @@ <h4><b>Priyanshu Khandelwal</b></h4>
       <a href="https://github.com/Masquerade0097" class="social-button fab fa-github"></a>
       <p>LINUX FOUNDATION CommunityBridge 2019 - CNCF</p>
     </div>
-    <div class="col-lg-3 centered">
+
```
Comment (vsvipul) [inline gsoc/index.html]: A new row should be created at this line. Check previous examples for how bootstrap works.

Comment (pranshukharkwal) [reply gsoc/index.html]: @vsvipul Added rows to the entire page. Please check

**Comment by vsvipul** [comment]
Check lines from 313-321 and 440 onwards. The whole structure alignment of the code should be same. Can you fix that too ? Rest LGTM.

**Comment by vsvipul** [comment]
thanks @pranshukharkwal  good work.

**Pull request merged**